### ISSUE TITLE
UpdateAddonsTaskBlocks on SSH

### ISSUE THREAD

**Issue by sminnee** [silverstripe/addons.silverstripe.org#171]
On a fresh server we get errors of this form:

```
The authenticity of host 'gitlab.com (52.167.219.168)' can't be established.
ECDSA key fingerprint is SHA256:HbW3g8zUjNSksFbqTiUWPWg2Bq1x8xdGUrliXFzSnUw.
Are you sure you want to continue connecting (yes/no)? 
```

For a background task, this isn't acceptable.

**Comment by sminnee** [comment]
A few further comments:

 * This may actually be BuildAddonsTask rather than UpdateAddonsTask
 * Fix is likely to add `-o StrictHostKeyChecking=no` to git's SSH command within composer's download maanger
 * This doesn't affect prod, just the migration to SSP (disposable machine images mean that the host keys aren't retained)

**Comment by sminnee** [comment]
It looks like setting the env var `GIT_SSH_COMMAND="ssh -o StrictHostKeyChecking=no"` will help

**Comment by robbieaverill** [comment]
@sminnee I think this is fixed with your work on the queued jobs implementation, yeah?

**Comment by sminnee** [comment]
I think that this was a bug that I developed in the course of building it, so I'll have to double check

**Comment by sminnee** [comment]
Yep fixed in 6d4cc87dca497d37adbfcd9bb6656c22757843ee
